reactDashboard/src/components/Footer/Footer.css

```diff
@@ -66,3 +66,11 @@
 footer {
   padding-bottom: 30px;
 }
+
+.darkmode footer {
+  background-color: #121212;
+}
+
+.darkmode header {
```

Comment:
What's this?

Reply:
It makes all the headers and footers dark. Previously, the individual elements were styled with the same background-color. I just combined them under header and footer. However, I have moved ".darkmode header" to "DashboardNavbar.css"